lambdawarmer/__init__.py

```diff
@@ -105,7 +106,7 @@ def _perform_fan_out_warm_up_calls(config, correlation_id, concurrency, lambda_c
     for i in range(1, concurrency):
         try:
             lambda_client.invoke(
-                FunctionName=LAMBDA_INFO['function_name'],
+                FunctionName='%s:%s' % (LAMBDA_INFO['function_name'], LAMBDA_INFO['function_version']),
```

Comment:
please use the newer string format syntax. you could do ...
```
'{function_name}:{function_version}'.format(**LAMBDA_INFO)
```

Reply:
fixed